flake.nix

```diff
@@ -14,16 +14,24 @@
 
   outputs = { self, flake-compat, nixpkgs, utils }: utils.lib.eachDefaultSystem (system: let
     pkgs = import nixpkgs { inherit system; };
-  in {
+  in rec {
+
+    packages.nixpkgs-hammer-ast-checks = pkgs.rustPlatform.buildRustPackage {
```

Comment:
Maybe we can just use `ast-checks` here. Otherwise hammer is the tool but the project is hammering.